packages/components/config/ember-try.js

```diff
@@ -22,15 +22,20 @@ module.exports = async function () {
         name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
+            '@ember/test-helpers': '^2.0.0',
+            'ember-a11y-testing': '^5.2.1',
+            'ember-cli': '~3.28.0',
+            'ember-qunit': '^6.0.0',
+            'ember-resolver': '^8.1.0',
             'ember-source': '~3.28.11',
```

Comment:
minor point/suggestion but I notice that TFC is still on 3.28.10. _Maybe_ it would be safer to pin to that version as the oldest possible scenario we should account for?

Reply:
[TFC is on `3.28.11`](https://github.com/hashicorp/atlas/blob/main/frontend/atlas/package.json#L144), Nomad is on `~3.28.10` but it likely resolves in 3.28.12 because of the `~`

Reply:
🤦 I can't read. Disregard 🥸